### ISSUE TITLE
Moving ACF Metaboxes

### ISSUE THREAD

**Issue by samjco** [elliotcondon/acf#756]
Trying to move my ACF metabox to side high but I cannot locate created metabox id.
For example:

```php
add_action('add_meta_boxes', function() {
   //ACF Metabox
  add_meta_box('acf-group_5c7e0ba222ee8', __('Project Actions'), '[UNKNOWN ID HERE]', null, 'side', 'high');
  add_meta_box('submitdiv', __('Publish'), 'post_submit_meta_box', null, 'side', 'low');
  add_meta_box('postimagediv', __('Featured Image'), 'post_thumbnail_meta_box', null, 'side', 'low');
});
```

**Comment by elliotcondon** [comment]
@samjco 

There is no "side high" location. There is a "side" location which you can select in the Field group settings.

**Comment by samjco** [comment]
@elliotcondon

I’m not sure I understand when you say “there is no side high”. Are you saying those parameters doesn’t exist is core? Or?

The “side” and “high”  params all seem to work when I move the built-in metaboxes. 
And even works in my example code above.

 However, I was trying to located the function Id param for an ACF metabox which I am able to get all the others that are builtin.

**Comment by elliotcondon** [comment]
Hi @samjco 

The available metabox locations are:
- normal
- side
- acf_after_title (high)

The callback function to replace '[UNKNOWN ID HERE]' is not a publicly available callback. This is all handled by ACF magic.

If you can explain what you are trying to do, without the code you are using, I will be happy to help advise on a solution :)

**Comment by samjco** [comment]
@elliotcondon

Thanks for explaining. You are only referring to ACF. I was going a custom route using add_meta_box function to move all the metaboxes around just the side area. 

I was trying to expose all metaboxes that are registered using global $WP_meta_boxes

I want to move all my ACF metaboxes ABOVE the WP builtin metaboxes programmatically.

**Comment by elliotcondon** [comment]
For that, you can use the "acf/input/meta_box_priority" filter which allows you to set the $priority of each ACF metabox. Setting this to "high" is the highest available priority.

Other than that, you can use JS to move DOM elements around if need be.

**Comment by samjco** [comment]
@elliotcondon 
Yeah, I was trying to not use js unless I have to. I don't like the element jumping to place after page loads. 

I was just referring to this:

Below are the other standard WordPress meta-box functions available while using the add_meta_box command (i.e. argument #1 = meta-box name or div name, argument #3 = function name) :

```php
post_submit_meta_box – submitdiv
post_tags_meta_box – tagsdiv-post_tag
post_categories_meta_box – categorydiv
post_excerpt_meta_box – postexcerpt
post_trackback_meta_box – trackbacksdiv
post_custom_meta_box – postcustom
post_comment_status_meta_box – commentstatusdiv
post_comment_meta_box – commentsdiv
post_slug_meta_box – slugdiv
post_author_meta_box
post_revisions_meta_box – revisionsdiv
page_attributes_meta_box – pageparentdiv
link_submit_meta_box – linksubmitdiv
link_categories_meta_box – linkcategorydiv
link_target_meta_box – linktargetdiv
link_xfn_meta_box – linkxfndiv
link_advanced_meta_box – linkadvanceddiv
post_thumbnail_meta_box – postimagediv
```

I am just wondering if ACF metabox are registered with a function name. If so, how can I locate it?

**Comment by samjco** [comment]
@elliotcondon can I use this like:

```
add_action('add_meta_boxes', function() {
   //ACF Metabox
add_meta_box('acf-group_5c7e0ba222ee8', __('Project Actions'), ACF_Form_Post::render_meta_box, null, 'side', 'high');
});
```

**Comment by samjco** [comment]
@elliotcondon 
I have just decided to use a nifty ACF addon plugin.
https://github.com/winteragency/acf-native-fields

But would await that update to for a static for ACF metaboxes callback. It just would kinda make sense to have this standard to WP core. Especially if a user wanted to move things around based on User Roles

**Comment by elliotcondon** [comment]
The only problem is that ACF uses a "generic callback" for all metaboxes and provides the $field_group as a parameter to this callback.

This would make it much more complicated than simply calling the `add_meta_box()` function.
Something to definitely think about, but am not sure about the reality of this change.
